Salary App/Features/SalaryForm/View/SalaryBreakDownViewController.swift

```diff
@@ -0,0 +1,261 @@
+//
+//  SalaryBreakDownViewController.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+ 
+
+import UIKit
+
+class SalaryBreakDownViewController: UIViewController {
+        
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    
+    private lazy var monthlyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryMonthlyLbl: UILabel = {
+        let salaryMonthlyLbl = UILabel()
+        salaryMonthlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryMonthlyLbl
+    }()
+    
+    private lazy var yearlyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryYearlyLbl: UILabel = {
+        let salaryYearlyLbl = UILabel()
+        salaryYearlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryYearlyLbl
+    }()
+    
+    private lazy var dailyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryDailyLbl: UILabel = {
+        let salaryDailyLbl = UILabel()
+        salaryDailyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryDailyLbl
+    }()
+    
+    private lazy var weeklyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryWeeklyLbl: UILabel = {
+        let salaryDailyLbl = UILabel()
+        salaryDailyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryDailyLbl
+    }()
+    
+    
+    private lazy var hourlyStackView: UIStackView = {
```

Comment:
Rather than repeating this for hourly, weekly, monthly etc... Can you build a single component that you can reuse? You're repeating a lot of the same views here.